src/styles/header.scss

```diff
@@ -0,0 +1,113 @@
+header {
+    position: fixed;
+    z-index: 999;
+    width: 100%;
+    height: 3rem;
+    min-height: 50px;
+    color: #fff;
+    background: rgba(0, 8, 81, 0.8);
+    border: 1px solid transparent;
+    border-radius: 0;
+
+    .input-group {
+      float: right;
+      width: 200px;
+      margin: 0.7rem 0 0;
+
+      @media (max-width: 720px) {
+        width: 150px;
+      }      
+
+      input {
+        height: 25px;
+        font-size: 0.8rem;
+      }
+
+      .bi {
+        color: #fff;
+      }
+
+      .search-link {
```

Comment:
taking into the consideration the nesting of the components, would be useful to use BEM Notation while naming styles